pkg/diskmaker/controllers/lvset/matcher.go

```diff
@@ -98,11 +100,12 @@ var matcherMap = map[string]func(internal.BlockDevice, *localv1alpha1.DeviceIncl
 			return false, fmt.Errorf("could not parse device size: %w", err)
 		}
 		greaterThanOrEqualToMin := true
-		if spec.MinSize != nil {
-			// quantity greater than min: -1
-			// quantity equal to min: 0
-			greaterThanOrEqualToMin = spec.MinSize.Cmp(quantity) <= 0
+		if spec.MinSize == nil {
+			spec.MinSize = &defaultMinSize
 		}
+		// quantity greater than min: -1
+		// quantity equal to min: 0
+		greaterThanOrEqualToMin = spec.MinSize.Cmp(quantity) <= 0
 
```

Comment:
How about also updating the rules so as if a partition has "boot" in substring, the partition is not selected? If they want to force use of such a partition they can always use `LocalVolume` object.

Reply:
added in `noBiosBootInPartLabel` rule above.